ecs-init/config/common.go

```diff
@@ -124,6 +124,15 @@ func DockerUnixSocket() (string, bool) {
 	return "/var/run", false
 }
 
+// CgroupMountpoint returns either the default cgroup mountpoint for the system or the mountpoint overidden at CGROUP_MOUNTPOINT
+func CgroupMountpoint() string {
+	if mount := os.Getenv("CGROUP_MOUNTPOINT"); mount != "" {
```

Comment:
Do we need an override?

We've talked a bit about configuring ecs-init before and erred on the side of not allowing configuration.  This makes ecs-init easier to understand (fewer possibilities).  If you're just using this for development, there's actually a [`development` build tag](https://github.com/aws/amazon-ecs-init/blob/bef8d94863624c3d045d264d873a72c934b43e74/ecs-init/config/development.go#L1) that can be used and helps separate development use-cases from production use-cases.

Reply:
We don't *need* it. Its just technically possible to override where cgroups live on the host system. I'd rather include the option.

Reply:
My preference would be to exclude it for now; we can always add it again later, but it's hard to remove later if we don't know whether people are using it or not.  When we do add configuration to ecs-init, I want to think a little harder about whether we should use environment variables or provide a different configuration mechanism.